code/game/objects/structures/mineral_doors.dm

```diff
@@ -271,14 +267,35 @@
 /obj/structure/mineral_door/update_icon()
 	icon_state = "[base_state][door_opened ? "open":""]"
 
+/obj/structure/mineral_door/proc/door_rattle()
+	playsound(src, rattlesound, 100)
+	var/oldx = pixel_x
+	animate(src, pixel_x = oldx+1, time = 0.5)
+	animate(pixel_x = oldx-1, time = 0.5)
+	animate(pixel_x = oldx, time = 0.5)
+
 /obj/structure/mineral_door/attackby(obj/item/I, mob/user)
+	user.changeNext_move(CLICK_CD_FAST)
```

Comment:
Why was this done? @EvenInDeathIStillServe?

Reply:
> Why was this done? @EvenInDeathIStillServe?

Because doors felt super nonresponsive to clicks, if you messed up on locking/unlocking.